docs/openapi-issuer.yaml

```diff
@@ -618,6 +645,9 @@ components:
             qr_code_url:
               type: string
               format: uri
+            notified_via_io_message:
```

Comment:
What about storing message id here instead, ie.

`...signatureRequest, 
  notifications: { 
     io_message_id: <id>
  }`

This would let (one day) the issuer recover the IO message notification status.

Reply:
It seems right!